icu_benchmarks/imputation/diffwave.py

```
@@ -356,10 +416,14 @@ def forward(self, input_data):
         h = noise
         skip = 0
         for n in range(self.num_res_layers):
-            h, skip_n = self.residual_blocks[n]((noise, conditional, diffusion_step_embed))
+            h, skip_n = self.residual_blocks[n](
```

Ruff (0.15.15)

[warning] Unpacked variable `h` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)